docs/TOC.yml

```diff
@@ -26,3 +26,8 @@
       items:
         - name: Customize controls
           href: user-interface/handlers/customize.md
+    - name: Device features
```

Comment:
https://github.com/dotnet/docs-maui/issues/16

The root node should be Platform integration, followed by an Essentials node, then Sensors.

Reply:
Ditto for the folder structure in the repo: docs/platform-integration/essentials/

Reply:
I'll follow up with a new PR to move all the files. With the TOC though, should we call out essentials as a section? What benefic does it provide to do 

**Device features** > **Essentials** > **Sensors** over
**Device features** > **Sensors**

The TOC doesn't need to logically follow the repo folder structure. I know the product will evolve with how essentials is presented to the developer, but to me, it doesn't make sense to call out essentials anymore. It's not a library or a product, you may need some config, but it's just using .NET MAUI to access to device features, essential or not. We can play with it.